charts/velero/README.md

```diff
@@ -93,12 +93,18 @@ helm upgrade vmware-tanzu/velero <RELEASE NAME> --reuse-values --set configurati
 ```
 ## Upgrading Chart
 
+### Upgrading to 7.0.0
```

Comment:
Do we have special notes such as not backward compatible compare to the previous version? If no, we could remove this one.